routers/repo/http.go

```diff
@@ -215,7 +215,10 @@ func HTTP(ctx *context.Context) {
 				// Check username and password
 				authUser, err = models.UserSignIn(authUsername, authPasswd)
 				if err != nil {
-					if !models.IsErrUserNotExist(err) {
+					if models.IsErrUserProhibitLogin(err) {
+						ctx.HandleText(http.StatusUnauthorized, "User is not permitted to login")
```

Comment:
StatusForbidden seems more legit in this case. http://www.dirv.me/blog/2011/07/18/understanding-403-forbidden/index.html
```suggestion
						ctx.HandleText(http.StatusForbidden, "User is not permitted to login")
```

Reply:
Damn you're right but it's just been merged!